lib/rules/require-camelcase-or-uppercase-identifiers.js

```diff
@@ -129,6 +129,17 @@ module.exports.prototype = {
                     return;
                 }
 
+                if (prevToken && prevToken.value === ':') {
```

Comment:
Of course! (I wasn't sure what the standard here was, didn't see it often).

I'm not sure if there's a better way but I'm not super deep into this code and have seen the `grandpa` pattern around. Please let me know if there is!